docs/identities/sign-in/native-app-auth.mdx

```diff
@@ -0,0 +1,59 @@
+---
+id: native-app-auth
+slug: native-app-login-authentication-ios-android-cli
+title: Implement native app login and registration
+sidebar_label: Native app authentication
+---
+
+# Native app login and registration
+
+This guide explains how to authenticate native apps. This includes:
+
+- Apps for mobile phones written in Swift, Android, React Native, Cordova, Ionic, and so on
+- Command line interface applications (CLIs)
+- Devices or apps without a browser
+
+:::info
+
+Ory's native app APIs do not use HTTP redirects and HTTP cookies but instead fully rely on bearer tokens, JSON and REST.
+
+:::
+
+These apps use the native API to sign in `createNativeLoginFlow()`, register `createNativeRegistrationFlow()`, change settings
+`createNativeSettingsFlow()`, and so on. Native apps receive an Ory session token instead of an Ory session cookie. The session
+token is used to look up the user.
+
+## Check session token
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import { getSdkUrl } from '@site/src/hooks'
+import CodeBlock from '@theme/CodeBlock'
+import CodeFromRemote from '@theme/CodeFromRemote'
+import reactNative from '!!raw-loader!../../../code-examples/identity/sign-in/check-session/js/react-native.tsx'
+
+<Tabs>
+  <TabItem value="react-native" label="React Native" default>
+    <CodeBlock language="tsx">{reactNative}</CodeBlock>
+  </TabItem>
+  <TabItem value="curl" label="curl">
+
+
+Read the [check login session](check-session-token-cookie-api) documentation for more details about authentication native apps.
+
+
+  </TabItem>
+</Tabs>
+```
```

Comment:
Yup, makes sense!

I used the non-file name here because it did not work when I used the filename - the build broke. It's the problem with tabitem indentation, apparently. But if we remove the tabs, it should work eitehr way.

Reply:
done